h2o-core/src/main/java/water/fvec/TransformWrappedVec.java

```diff
@@ -36,16 +37,16 @@
 
   private final Key<Vec>[] _masterVecKeys;
   private transient Vec[] _masterVecs;
-  private final AstRoot _fun;
+  private final AstPrimitive _fun;
 
-  public TransformWrappedVec(Key key, int rowLayout, AstRoot fun, Key<Vec>... masterVecKeys) {
+  public TransformWrappedVec(Key key, int rowLayout, AstPrimitive fun, Key<Vec>... masterVecKeys) {
```

Comment:
They are conceptually different entities: `AstExec` is a bound execution context, i.e. a function together with the list of arguments to be passed to that function. 
On the other hand, `AstPrimitive` (which ought to be renamed into `AstFunction`) is an unbound function, i.e. function that expects some arguments to be passed to it before it can be executed.